scripts/build-css-arm64v8.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Could you remove this file & change the shebang in the existing `build-css` script? I figure it is best to just keep one around

Reply:
If you change it on the other script, the other builds will need an /bin/sh shell to be able to run this otherwise you'll get errors. It can be changed but this would need to be validated first.

Reply:
I just pushed some changes to this script to master that should make it compliant to both shells.

Reply:
Could you pull and then change this to a single script?

Reply:
I just added commits to this pull, the real fix here is to have `Dockerfile` use alpine images as well. Since it was just the normal `node:14` , that image (based on debian) uses a bash shell not a bourne shell that supports `#!/bin/sh`. You'd have to include an install process to install the bourne shell in the `node:14` image to make it compatible if you didn't want to use alpine.

However, now there's a new issue. Build was failing on CentOS 8 with `The command '/bin/sh -c npm install --silent && ./scripts/build-css.sh' returned a non-zero code: 1` error. Similar to the error I saw on arm64v8 when not using the correct architecture images. In this case it appeared to be caused by the `#!/bin/sh` in the file. To make it work I change the first build to use the `node:14-alpine3.13` image to match the arm64 version. However the error is still happening, but it's not failing on the script it's failing as part of the install of tailwindcss (see output below).

```
[root@usauto01 patrol]# docker build .
Sending build context to Docker daemon  1.885MB
Step 1/23 : FROM node:14-alpine3.13
14-alpine3.13: Pulling from library/node
ca3cd42a7c95: Pull complete
e42b8ae7c9dc: Pull complete
e5a38fa943ad: Pull complete
6cd7f2a0541e: Pull complete
Digest: sha256:4ffbef007b0214706fb8ec92353ccd5b0a12d9d1522e0f2c5e0a8bde3f9d8985
Status: Downloaded newer image for node:14-alpine3.13
 ---> 4355e9cbf49f
Step 2/23 : WORKDIR /app
 ---> Running in c23965ef68e2
Removing intermediate container c23965ef68e2
 ---> b7916af48404
Step 3/23 : COPY package.json .
 ---> 7e1ecb483e85
Step 4/23 : COPY package-lock.json .
 ---> 25cbdf7a013a
Step 5/23 : COPY scripts scripts
 ---> 726f91f35c01
Step 6/23 : COPY index.html .
 ---> f7c296061960
Step 7/23 : COPY tailwind.config.js .
 ---> 542925c3d218
Step 8/23 : COPY postcss.config.js .
 ---> 7069d35cf4d5
Step 9/23 : RUN npm install --silent && ./scripts/build-css.sh
 ---> Running in 40717bbc6428
The command '/bin/sh -c npm install --silent && ./scripts/build-css.sh' returned a non-zero code: 1
[root@usauto01 patrol]# docker run -it 7069d35cf4d5 sh
/app # npm install

/app # npm install
[       ...........] / extract:tailwindcss: verb lock using /root/.npm/_locks/staging-bd3355550c5a469e.lock for /app/node_modules/.staging
```

It stalls out on that stage of the NPM install and eventually just quits.

Reply:
Have you tried running `ghcr.io/karimsa/patrol:unstable-arm64v8` image on your raspberry pi? Could you try and let me know if that image works?

Reply:
Current unstable-arm64v8 build is failing to run on RPi4.

`standard_init_linux.go:211: exec user process caused "exec format error"`

Looking at your ci.yml file for github actions, I see some problems which would cause this.

1. The unstable build job isn't targeting the Dockerfile.arm64v8 file.e
```
    - name: Build arm64v8 image
      run: docker build -t ghcr.io/karimsa/patrol:unstable-arm64v8 .
```

change to 

```
    - name: Build arm64v8 image
      run: docker build -t ghcr.io/karimsa/patrol:unstable-arm64v8 -f Dockerfile.arm64v8 .
```

2. While you can make the changes above, in my own testing...github actions will most likely fail to successfully build an ARM64v8 image without using docker buildx in your actions. Without using docker buildx in your action CI cannot successfully emulate a ARM64 environment and produce an image.

Reply:
Good catch! Can you try again? I updated the CI to build the arm image this time.

Reply:
Nice! Looks to be working as expected. Container runs without issue, web interface is reachable and it's running without issue :)

Reply:
Awesome! Should I go ahead and close this PR then?

I'm hoping that the CI building + releasing the RPi image is more convenient for you! And ideally, I'd like to leave building directly on the RPi out of the scope of this project, if possible, since it seems like it would be much more complex.

Reply:
Yeah this can be closed, things are looking good! 👍